src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$card-width: 200px;
+$img-width: calc(($card-width / 100) * 80);
+$img-height: calc(($card-width / 100) * 67);
+$card-title-width: calc(($card-width / 100) * 83);
+$card-product-witdth: calc(($card-width / 100) * 49);
+$stars-width: $card-title-width;
+$card-price-width: $card-title-width;
+$card-button-width: $card-title-width;
```

Comment:
Very complex formulas for width, you need to use 100% width and set paddings for the card itself